cluster/charts/rook-ceph/values.yaml.tmpl

```diff
@@ -21,6 +21,8 @@ resources:
     cpu: 100m
     memory: 128Mi
 
+nodeSelector:
+
```

Comment:
1) will this allow multiple lines if you want multiple conditions on the nodeSelector? Or is it restricted to a single condition?
2) want to add an example that is commented out so it is clear what format is needed?